debian/control

```diff
@@ -0,0 +1,16 @@
+Source: youtube-dl-gui
+Section: web
+Priority: optional
+Maintainer: nodiscc <nodiscc@gmail.com>
+Build-Depends: debhelper (>= 9), python-all
```

Comment:
There is also the **twodict** requirement but there is no debian package for it.
Maybe we should also add **python-pip** since the easiest way to install **twodict** is the PyPi one.